src/Ilios/WebBundle/Controller/IndexController.php

```diff
@@ -4,16 +4,25 @@
 
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Response;
+use Ilios\CliBundle\Command\UpdateFrontendCommand;
+
```

Comment:
It's worse than the line - that service isn't even used in the file.  I just pushed an updated one.